### DISCUSSION TITLE
Incorrect search by product SKU in the admin hub

### DISCUSSION THREAD

**Discussion by devmansurov** [lunarphp/lunar#75] [Q&A]
My products SKU starts from `3361` in my store like this:

![image](https://user-images.githubusercontent.com/51310839/148977588-26d766d2-6a44-4dd8-a90b-946e800407b9.png)

When I enter search query `3361` from SKU don't get all products.

To test you can see my demo store here:

https://candyshop.na4u.ru/hub

E-mail: otabek.902777653@gmail.com
Password: 00000000

Scout driver: mysql

**Comment by glennjacobs** [comment]
I presume you mean `3611`. Have you tried the Meilisearch driver? MySQL search driver isn't really for production and is quite limited.

**Comment by devmansurov** [comment]
> tried the Meilisearch driver

I tried the Meilisearch driver but it turns out that the scout does not index product attributes, which is why it is not possible to search by product attributes such as SKU.

I tried run this command to reindex:

`php artisan getcandy:meilisearch:setup`

But doesn't work

**Comment by alecritson** [comment]
When indexing with Meilisearch (or anything other than MySQL) we index the skus into a field `skus` on the index, which should then be searchable.

When you run the set up command it won't reindex everything as this should be explicitly done by the developer (to stop any unwanted changes to the documents)

You can import all products by running the scout command

```bash
php artisan scout:import GetCandy\\Models\\Product
```
